src/core/containers/LoginRequired/index.js

```diff
@@ -7,7 +7,7 @@ import LoginPage from 'core/components/LoginPage';
 
 export function mapStateToProps(state) {
   return {
-    authenticated: !!state.api.token,
+    authenticated: !!state.user.id,
```

Comment:
It looks like you didn't have to update any tests for this which is a red flag. Can you try to add test coverage for it?

Reply:
It is actually covered by the change in `dispatchSignInActions()` 😉

Reply:
oh I see, that's cool. The helper is doing its job well.

You could edit the specs in `TestLoginRequired` but it's not a big deal. I would propose the following changes:

* old: it sets authenticated to true when there is a token
  * new: it sets authenticated to true
* old: it sets authenticated to false when there is not a token
  * new: it sets authenticated to false